### ISSUE TITLE
Arabic Translation Header Should Be RTL

### ISSUE THREAD

**Issue by wisn** [github/training-kit#561]
## Overview
**TL;DR**
The main body of the Arabic translation is right-to-left (RTL) aligned. So, the header should be RTL as well.

#### Screenshots
![screenshot from 2017-08-23 00-29-12](https://user-images.githubusercontent.com/8147926/29578684-5b43f3c6-879a-11e7-81e2-e5f2483be17a.png)

### How to Contribute
- [x] Comment on this issue if you'd like to work/collaborate on it! 
- [x] Fork the [repository](https://github.com/github/training-kit).
- [x] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com{{site.baseurl}}/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [x] Push your changes to your repository. 
- [x] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [x] Run `script/setup`
  - [x] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000{{site.baseurl}}`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!

**Comment by hollenberry** [comment]
@wisn You're absolutely right! Thank you for bringing this up! 

Would you be interested in opening up a PR to help us make the change?

**Comment by wisn** [comment]
@hollenberry Actually, I'm planning to fix markdown on French translation first.

![screenshot from 2017-08-23 00-38-08](https://user-images.githubusercontent.com/8147926/29578966-739feec4-879b-11e7-8822-546f0d4c6f61.png)

**Comment by wisn** [comment]
This issue has been solved in #566.
